java-core/solid/fruit-shop-tests.md

```diff
@@ -61,8 +61,14 @@ public void getReport_Ok() {
     Assert.assertEquals(expected, actual);
 }
 
-@After
+@AfterEach
 public void afterEachTest() {
     Storage.storage.clear();
 }
```

Comment:
I think some student had a problem with `after each` method, and it was solved by swapping to the `before each`, not sure what caused that problem, but maybe it is worth changing here too? Just in case